README.md

```diff
@@ -2,21 +2,49 @@
 
 Creates a new tag using the format `Y.M.D` (using [daily-version](https://github.com/fregante/daily-version)), but only if `HEAD` isn’t already tagged.
 
-Ideally used on schedule, but you could also change the suggested condition to only make it run on `master`
+Ideally used on schedule, but you could also change the suggested condition to only make it run on `master`.
 
-# Usage
+## Usage
 
-See [action.yml](action.yml)
+It expects `git` to be configured to push to the same repo, perhaps via [fregante/setup-git-token](https://github.com/fregante/setup-git-token)
 
-Basic:
+See [action.yml](action.yml)
 
 ```yaml
+  Version:
     steps:
     - uses: actions/checkout@v2
-    - uses: fregante/setup-git-token@v1 # This will allow saving the generated tag
+    - uses: fregante/setup-git-token@v1 # Allows pushing the generated tag to the repo
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
     - name: Create tag if necessary
-      if: github.event_name == 'schedule' # This ensures it only runs on a schedule
+      if: github.event_name == 'schedule' # Ensures it only runs on a schedule
       uses: fregante/daily-version-action@v1
 ```
+
+To check if any new version has been created in other jobs, use [`job.outputs`](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjobs_idoutputs) object to see if `created` property is set. This property would not be set if any new versions are created.
+
+```yaml
+  Build:
+    needs: Version
+    if: github.event_name == 'push' || needs.Version.outputs.created
```

Comment:
Let's keep it simple
```suggestion
    - name: 'Create tag if necessary'
      id: version
      uses: fregante/daily-version-action@v1
    - name: 'Created?'
      if: steps.version.outputs.created
      runs: echo 'Created!' ${{ steps.version.outputs.version }}
```

Untested though.

Reply:
How 'bout adding `steps` at the start. For verbosity.

Reply:
Sure